backend/test_missing_endpoints.py

```
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+"""
+Test script to check which frontend API endpoints are working
+"""
+
+import asyncio
+import asyncpg
+import os
+from datetime import datetime
```

Ruff (0.12.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)